pkg/storage/registry/spaces/spaces.go

```diff
@@ -52,17 +52,31 @@ func init() {
 	pkgregistry.Register("spaces", NewDefault)
 }
 
-type provider struct {
-	Mapping           string            `mapstructure:"mapping"`
-	MountPath         string            `mapstructure:"mount_path"`
-	Aliases           map[string]string `mapstructure:"aliases"`
-	AllowedUserAgents []string          `mapstructure:"allowed_user_agents"`
-	PathTemplate      string            `mapstructure:"path_template"`
-	template          *template.Template
+type spaceConfig struct {
+	// MountPoint determines where a space is mounted. Can be a regex
+	// It is used to determine which storage provider is responsible when only a path is given in the request
+	MountPoint string `mapstructure:"mount_point"`
+	// PathTemplate is used to build the path of an individual space. Layouts can access {{.Space...}} and {{.CurrentUser...}}
+	PathTemplate string `mapstructure:"path_template"`
```

Comment:
Just for clarification: Will `MountPoint` always be a prefix of `PathTemplate`?

Reply:
AFAICT yes. But we cannot just use one config option because the mount_point might be a regex.